### PR TITLE
fix: can not pod install with react-native-gesture-handler

### PR DESCRIPTION
**Pull request by hangnguyensaigontech**
## Bugs
- Root cause: Pod Install Failed when install rn-base-component in rn-base-project
- Solution: move react-native-gesture-handler into peerDependencies instead of dependencies to resolve conflict versions

## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

### CODE REVIEW

**Review by chungnguyenhoang — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -87,15 +87,17 @@
     "styled-components": "^5.3.6",
     "ts-jest": "^29.0.5",
     "ts-node": "^10.9.1",
-    "typescript": "^4.5.2"
+    "typescript": "^4.5.2",
+    "react-native-gesture-handler": "^2.9.0"
```
Comment (baonguyenhsts) [inline package.json:91]: the version should be 2.13.3

**Review by baonguyenhsts — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (baonguyenhsts) [reply package.json:91]: Also, we need to make sure the version in `rn-base-component` and `rn-base-project/rn-base-project-typescript` is the same

Comment (hangnguyensaigontech) [reply package.json:91]: Hi @baonguyenhsts  Hmm I understand but we already included `^` which means as long as we yarn install, it will get latest from version 2.x.x so actually we don't need to change it everytime it have new version with 2.x.x

Comment (baonguyenhsts) [reply package.json]: Okay, I just need to ensure that their versions are identical.

**Review by huydosgtech — APPROVED** [review]

**Review by baonguyenhsts — APPROVED** [review]
